src/test/java/de/zalando/typemapper/postgres/PgSerializerTest.java

```diff
@@ -61,8 +58,8 @@ public PgSerializerTest(final Object objectToSerialize, final Object expected) {
     public static Collection<Object[]> generateData() throws SQLException {
         return Arrays.asList(
                 new Object[][] {
-                    {new Date(112, 11, 1, 6, 6, 6), Pattern.compile("2012-12-01 06:06:06.000000 [+-]?\\d{2}:\\d{2}:00")},
-                    {new Date(112, 9, 1, 6, 6, 6), Pattern.compile("2012-10-01 06:06:06.000000 [+-]?\\d{2}:\\d{2}:00")},
```

Comment:
Do you know why the milliseconds are dropped here? should we add non-zero millisecond containing test?